web/docs/use/query/README.md

```diff
@@ -1,129 +1,228 @@
 # Query
 
-import MissingDocumentation from '@site/presets/MissingDocumentation.md';
+Querying data from VAST (aka *exporting*) involves spinning up a VAST client
+that executes a query expression. In the following, we assume that you [set up a
+server](/docs/use/run) listening at `localhost:42000`.
 
-<MissingDocumentation/>
+To run a query, you need to:
 
-## Choose an export format
-
-<MissingDocumentation/>
-
-### Zeek
+1. know what to look for
+2. express what you want
+3. decide in what format to show the result
 
-<MissingDocumentation/>
+Let's go through each of these steps.
 
-#### Broker
+## Decide what to query
 
-The `broker` export command sends query results to Zeek via the
-[Broker](https://github.com/zeek/broker) communication library.
+To figure out what you can query, VAST offers
+[introspection](/docs/use/introspect) via the `show` command.
 
-Broker provides a topic-based publish-subscribe communication layer and
-standardized data model to interact with the Zeek ecosystem. Using the `broker`
-writer, VAST can send query results to a Zeek instance. This allows you to write
-Zeek scripts incorporate knowledge from the past that is no longer in Zeek
-memory, e.g., when writing detectors for longitudinal attacks.
-
-To export a query into a Zeek instance, run the `broker` command:
+Use `show schemas` to display the layout of all types:
 
 ```bash
-# Spawn a Broker endpoint, connect to localhost:9999/tcp, and publish
-# to the topic `vast/data` to send result events to Zeek.
-vast export broker <expression>
+vast show schemas --yaml
 ```
 
-To handle the data in Zeek, your script must write a handler for the following
-event:
-
-```zeek
-event VAST::data(layout: string, data: any)
-  {
-  print layout, data; // dispatch
-  }
+In case you ingested [Suricata](/docs/understand/format/suricata) data, this may
```

Comment:
```suggestion
In case you ingested [Suricata](/docs/understand/formats/suricata) data, this may
```

Is our dead link detection broken?

Reply:
No, it only kicks in for `docusaurus build`. When building locally, that's rarely what we use.